File: backend/app/models/common/info_response.go
package common

type InfoResponse struct {
	Success bool        `json:"success"`
	Message string      `json:"message,omitempty"`
	Data    interface{} `json:"data,omitempty"`
}

//func SuccessResponse(data any, msg string) *InfoResponse {
//	return &InfoResponse{
//		Success: true,
//		Message: msg,
//		Data:    data,
//	}
//}

File: frontend/lib/models/response/common/info_response.dart
class InfoResponse {
  final bool success = true;
  final Map<String, dynamic>? data;
  final String? message;

  InfoResponse({
    this.data,
    this.message,
  });

  InfoResponse.fromJson(Map<String, dynamic> json)
      : data = json["data"],
        message = json["message"];
}
